src/Messaging/src/RabbitMQ/Attributes/DeclareAnonymousQueueAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Steeltoe.Messaging.Rabbit.Core;
+using System;
+
+namespace Steeltoe.Messaging.Rabbit.Attributes
```

Comment:
Should this namespace be RabbitMQ?

Reply:
Ya ... there is a task to change the namespaces .. just not done yet